### ISSUE TITLE
Incorrect Yoda condition error

### ISSUE THREAD

**Issue by Chouby** [WordPress-Coding-Standards/WordPress-Coding-Standards#1036]
Hi,

With WPCS 0.11, I get a Yoda condition error on the line: 
```php
$sample = false !== strpos( $link, '%pagename%' );
```
I believe that it is a false positive and looks to be the same issue as #509
I don't get any error if I modify the code to:
```php
$sample = ( false !== strpos( $link, '%pagename%' ) );
```

**Comment by JDGrimes** [comment]
Thanks for the bug report @Chouby. I've opened #1037 to fix this issue.

**Comment by Chouby** [comment]
Thanks :)
